timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -213,45 +206,40 @@ def fit(
             ``self.validation_splitter``. If ``tuning_data`` is provided, ``self.validation_splitter`` will be ignored.
             See the description of ``validation_splitter`` in the docstring for
             :class:`~autogluon.timeseries.TimeSeriesPredictor` for more details.
 
-            # TODO: Mention that this is a good default that users should not override?
-            # TODO: Link to advanced tutorial explaining how to do more fancy validation / how validation works?
-        time_limit : int, default = None
+            Leaving this argument empty and letting AutoGluon automatically generate the validation set from
+            ``train_data`` is a good default.
+        time_limit : int, optional
             Approximately how long :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will run (wall-clock time in
             seconds). If not specified, :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will run until all models
             have completed training.
         presets : str, default = "medium_quality"
             Optional preset configurations for various arguments in
             :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit`.
 
-            # TODO: Better explanation of how to choose presets
             Can significantly impact predictive accuracy, memory footprint, inference latency of trained models,
             and various other properties of the returned predictor. It is recommended to specify presets and avoid
             specifying most other :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` arguments or model
             hyperparameters prior to becoming familiar with AutoGluon. For example, set ``presets="best_quality"``
             to get a high-accuracy predictor, or set ``presets="low_quality"`` to get a toy predictor that
             trains quickly but lacks accuracy.
-            # TODO: This looks like a legacy sentence, remove it? Are we referring to the `hyperparameters` kwarg here?
             Any user-specified arguments in :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will
             override the values used by presets.
 
-            # TODO: Should we even mention low_quality_hpo?
-            Available presets are "best_quality", "high_quality", "good_quality", "medium_quality", "low_quality",
-            and "low_quality_hpo". Details for these presets can be found in
-            ``autogluon/timeseries/configs/presets_configs.py``. If not provided, user-provided values for other
-            arguments (specifically, ``hyperparameters`` and ``hyperparameter_tune_kwargs`` will be used (defaulting
-            to their default values specified below).
+            Available presets are "best_quality", "high_quality", "good_quality", "medium_quality", and "low_quality".
+            Details for these presets can be found in ``autogluon/timeseries/configs/presets_configs.py``. If not
```

Comment:
Is this the case? If not provided, doesn't it "default" to medium_quality?

Reply:
As far as I understand, currently the default is `presets = None, hyperparameters = None`. In this case `hyperparameters` are set to `"default"`, so, technically, the presets are not used and exactly what's described in this docstring happens. This, however, is equivalent to setting `presets = "medium_quality"`. I think this is the same logic as in tabular.

Does this make sense, or do you think we can improve the phrasing here?

Reply:
The correct signature here would be `presets : str, optional`

Reply:
I see. I agree with `presets : str, optional`. Thanks!